### DISCUSSION TITLE
Using Headless UI with UnoCSS

### DISCUSSION THREAD

**Discussion by I7T5** [unocss/unocss#2781] [Q&A]
I'm a web hobbyist. Really like UnoCSS and would like to use it with Nuxt 3 for a personal project. The only thing is that with UnoCSS I don't have many component libraries to choose from (Onu, Anu-vue mentioned in #1179), especially those copy-and-paste ones like [shadcn/ui](https://github.com/shadcn/ui), which is only for React frameworks. 

Then I came across TailwindCSS's [Headless UI](https://headlessui.com/vue). I'm wondering if I can use HeadLess UI **without installing the library** since UnoCSS is compatible with TailwindCSS. I don't want to have so many libraries in my project. 

I'm mostly interested in using the Combobox, though I might need Disclosure, Dialog, and Transition.

**Comment by QaahirStewart** [comment]
you can use https://ui.nuxtlabs.com/getting-started

**Comment by john-prutton** [comment]
For those who are looking for UnoCSS + [shadcn-ui](https://ui.shadcn.com/) for React, have a look at [unocss-preset-shadcn](https://github.com/fisand/unocss-preset-shadcn/). I'm currently trying to get it to work although it seems the maintainer has a demo. Let me know if it works for you by reacting to this comment.

**Comment by stephenhebert** [comment]
I use UnoCSS with Headless UI without Tailwind and you can too.
